<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="welcome_message">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>821</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добро пожаловать в SQLAdmin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="opendb_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createdb_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать базу данных</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
